admin_manual/configuration/user_auth_ldap_cleanup.rst

```diff
@@ -0,0 +1,69 @@
+=================
+LDAP User Cleanup
+=================
+
+LDAP User Cleanup is a new feature in the ``LDAP user and group backend`` 
+application. LDAP User Cleanup is a background process that automatically 
+searches the ownCloud LDAP mappings table, and verifies if the LDAP users are 
+still available. Any users that are not available are marked as ``deleted`` in 
+the ``oc_preferences`` database table. Then you can run a command to display 
+this table, displaying only the users marked as ``deleted``, and then you have 
+the option of removing their data from your ownCloud data directory.
+
+These items are removed upon cleanup:
+
+* Local ownCloud group assignments
+* User preferences (DB table ``oc_preferences``)
+* User's ownCloud home folder
+* User's corresponding entry in ``oc_storages``
+
+There are two prequisites for LDAP User Cleanup to operate:
+
+1. Set ``ldapUserCleanupInterval`` in ``config.php`` to your desired check 
+   interval in minutes. The default is 51 minutes.
+
+2. All configured LDAP connections are enabled and operating correctly. As users 
+   can exist on multiple LDAP servers, you want to be sure that all of your 
+   LDAP servers are available so that a user on a temporarily disconnected LDAP 
+   server is not marked as ``deleted``.
+   
+The background process examines 50 users at a time, and runs at the interval you 
+configured with ``ldapUserCleanupInterval``. For example, if you have 200 LDAP 
+users and your ``ldapUserCleanupInterval`` is 20 minutes, the process will 
+examine the first 50 users, then 20 minutes later the next 50 users, and 20 
+minutes later the next 50, and so on.
+
+This is done with the ``occ`` command, which must be run as your HTTP user. The 
```

Comment:
Confusing. The background process as mentioned before is triggered by cron (maybe hint to set up a cronjob?). But this section is about managing the findings. I.e. starting it with "This is done with…" sounds wrong.